oaknational/Oak-Web-Application
# LESQ-871: Update Downloads Queries

## Description

Music year: 1968

- Refactors downloads query and Canonical downloads query into one file
- Separate construct functions for both downloads and canonical downloads
- Creates downloads utils file and test file for helper functions 
- N.B. pupil squad files also amended

## Issue(s)

Fixes #LESQ-871

## How to test

1. Go to https://deploy-preview-2472--oak-web-application.netlify.thenational.academy
2. Navigate to any downloads page
3. There should be no difference in the functioning of the page

## Checklist
